### ISSUE TITLE
Mailnag does not want to check mail accounts

### ISSUE THREAD

**Issue by rimvydukas** [pulb/mailnag#185]
Hi,
Problem is that mailnag does not check for a new mails when computer is restarted. An excerpt from log:
Apr 24 11:18:47 aladdin-mint mailnag[1648]: INFO (2019-04-24 11:18:47): Successfully enabled plugin 'userscriptplugin'.
Apr 24 11:18:47 aladdin-mint mailnag[1648]: INFO (2019-04-24 11:18:47): Successfully enabled plugin 'dbusplugin'.
Apr 24 11:18:47 aladdin-mint mailnag[1648]: INFO (2019-04-24 11:18:47): Checking 1 email account(s).
It seems that app is working but I get no notifications about any new mails:/ app starts to work when I enter app's properties, select mail account and expand folders. When I do that app can work for a day or even longer but it stops again after some time:( How can I fix this?

Linux Mint 19.1, mailnag is the latest version from the PPA.

**Comment by gitsnow** [comment]
I'm having the exact same issue using Mailnag 1.2.1-1.1 on Ubuntu 18.10.  If I force a Check Mail than it successfully finds new mail, but it is not checking on its own on a regular schedule.

**Comment by pulb** [comment]
@rimvydukas: what's your mail provider? Can you post your ~/.mailnag/config/mailnag.cfg?
@gitsnow : Please run `journalctl -b _COMM=mailnag` in a terminal when the problem occurs and post the output here

**Comment by gitsnow** [comment]
> @gitsnow : Please run `journalctl -b _COMM=mailnag` in a terminal when the problem occurs and post the output here

Thanks for following up.  I ended up adding your Ubuntu repository ppa:pulb/mailnag and upgrading to version 1.3.0-0~ubuntu18.10.1 and it seems to be checking mail on its own again.  I'll let you know if the problem reoccurs.

**Comment by rimvydukas** [comment]
> @rimvydukas: what's your mail provider? Can you post your ~/.mailnag/config/mailnag.cfg?

Mail provider is Gmail. Config output:

`[spamfilterplugin]
filter_text = newsletter, viagra

[core]
imap_idle_timeout = 10
enabled_plugins = dbusplugin, userscriptplugin
poll_interval = 10
credentialstore = auto
connectivity_test = auto
autostart = 1

[soundplugin]
soundfile = mailnag.ogg

[libnotifyplugin]
notification_mode = 3
max_visible_mails = 10

[account1]
enabled = 1
type = imap
name = user@mydomain.info (Gmail)
user = user@mydomain.info
password = password
server = imap.gmail.com
port = 993
ssl = 1
imap = 1
idle = 1
folder = ["Inv/P2P paskolos/Env"]

[userscriptplugin]
script_file = /home/user/Apps/mailnags/important-env-mail
`
And as I said - if I enter mailnag config, try to edit account and expand gmail folders - mailnag works for some time but stops after few days or even earlier:/

**Comment by pulb** [comment]
@rimvydukas: Can you disable the userscript plugin and remove the Inv/P2P paskolos/Env folder, just to see if it makes a difference?

**Comment by gitsnow** [comment]
It looks like I spoke to soon.  Mailnag stopped checking mail after a day or so.  
I ran journalctl -b _COMM=mailnag, but I don't see any output after the date I upgraded to 1.3.0.0.

**Comment by rimvydukas** [comment]
> @rimvydukas: Can you disable the userscript plugin and remove the Inv/P2P paskolos/Env folder, just to see if it makes a difference?

But my main wish is to monitor this folder and to be able to run script. And as I can see - I'm not the only one with this problem.

**Comment by pulb** [comment]
Yes, I know that disabling your script and folder is no solution, I'm just trying to pinpoint the problem.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by rimvydukas** [comment]
> Yes, I know that disabling your script and folder is no solution, I'm just trying to pinpoint the problem. -- Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

Ok, I disabled script and selected libnotify option. Will monitor thing. As regarding folder unselect. If I'll unselect this folder, what monitor then? Inbox? Because I have only two folders selected. Inbox is not in this list.

**Comment by pulb** [comment]
Yes, if no folders are specified, it monitors just the inbox. If the problem does not occur with this configuration, please re-enable your script and check if it still works.

**Comment by rimvydukas** [comment]
> Yes, if no folders are specified, it monitors just the inbox. If the problem does not occur with this configuration, please re-enable your script and check if it still works.

Ok, did exactly as you said and after the few days - the same story:/ I noticed one strange thing though...
We have 2019.05.14 when I saw that mailnag is not working. But when I look at the log file, last entries are only from 05.11:
May 11 09:07:48 user-mint mailnag[19751]: INFO (2019-05-11 09:07:48): Checking 1 email account(s).
May 11 09:17:50 user-mint mailnag[19751]: INFO (2019-05-11 09:17:50): Checking 1 email account(s).
May 11 09:19:57 user-mint mailnag[19751]: INFO (2019-05-11 09:19:57): Checking 1 email account(s).

But from the other side I see mailnag process runing:

user  19751  0.0  0.2 732852 39720 ?        Sl   May10   1:38 /usr/bin/python /usr/bin/mailnag

So I'm waiting for your further instructions.

**Comment by pulb** [comment]
Possibly related: https://github.com/pulb/mailnag/issues/186, https://github.com/pulb/mailnag/issues/188
